src/pkg/blob/dao/dao_test.go

```diff
@@ -267,13 +267,44 @@ func (suite *DaoTestSuite) TestExistProjectBlob() {
 func (suite *DaoTestSuite) TestDeleteProjectBlob() {
 	ctx := suite.Context()
 
-	projectID := int64(1)
-	blobID := int64(1000)
+	digest := suite.DigestString()
+	blobID, err := suite.dao.CreateBlob(ctx, &models.Blob{Digest: digest})
+	suite.Nil(err)
```

Comment:
Ideally, you should also test the case when input project ID doesn't exist.

Reply:
Test case when input the projectID which doesn't have records in the `project_blob` table was added.